insights/parsers/tuned_conf.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module contains following parser:
+
+TunedConfIni - file ``/etc/tuned.conf``
+---------------------------------------
+"""
+
+from .. import parser, Parser, get_active_lines
```

Comment:
Better uniform the usage of the `from` path in one single python parser file.
```
from insights.parsers import parser, Parser, get_active_lines
```

Reply:
Done